ogcapi-types/src/features/feature.rs

```diff
@@ -69,6 +79,23 @@ pub struct Feature {
     #[cfg(feature = "stac")]
     #[serde(default)]
     pub assets: HashMap<String, crate::stac::Asset>,
+    #[cfg(feature = "movingfeatures")]
+    #[serde(default, serialize_with="crate::common::serialize_interval", skip_serializing_if = "Vec::is_empty")]
+    /// Life span information for the moving feature.
+    /// See [MF-Json 7.2.3 LifeSpan](https://docs.ogc.org/is/19-045r3/19-045r3.html#time)
+    pub time: Vec<[Option<DateTime<Utc>>; 2]>,
+    #[cfg(feature = "movingfeatures")]
+    #[serde(default)]
+    pub crs: Option<Crs>,
```

Comment:
If missing means default crs in the spec, we can drop the Option. Same for trs below.